pkg/asset/internal/templates.go

```diff
@@ -419,4 +420,42 @@ spec:
     port: 53
     protocol: TCP
 `)
+	EtcdOperatorTemplate = []byte(`apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: etcd-operator
+  namespace: kube-system
+  labels:
+    k8s-app: etcd-operator
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: etcd-operator
+    spec:
+      containers:
+      - name: etcd-operator
+        image: quay.io/coreos/etcd-operator
+        env:
+        - name: MY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+`)
+	EtcdSvcTemplate = []byte(`apiVersion: v1
+kind: Service
+metadata:
+  name: etcd-service
+  namespace: kube-system
+spec:
+  selector:
+    app: etcd
+    etcd_cluster: etcd-cluster
+  clusterIP: 10.3.0.20
```

Comment:
I'm fine changing to "15" here.
Just to clarify, we don't want to explicitly support any specific thing here in bootkube.
If there is concerns from integrations, we should make this configurable.